etc/config/c.amazon.properties

```diff
@@ -1,6 +1,6 @@
 compilers=&cgcc86:&cclang:&armcclang32:&armcclang64:&cmosclang-trunk:&rvcclang:&wasmcclang:&ppci:&cicc:&cicx:&ccl:&ccross:&cgcc-classic:&cc65:&sdcc:&ctendra:&tinycc:&zigcc:&cproc86:&chibicc:&z88dk:&compcert:www.godbolt.ms@443:godbolt.org@443/winprod:&movfuscator:&lc3:&upmem-clang
 defaultCompiler=cg132
-demangler=/opt/compiler-explorer/gcc-13.1.0/bin/c++filt
+demangler=/opt/compiler-explorer/clang-trunk/bin/llvm-cxxfilt
```

Comment:
should revert this too

Reply:
No problem - but shouldn't this be just `c++filt` without the path? (I mena, rely on the /usr/bin/ symlink?)

Reply:
No, this is for all the gcc's for the C compilers, the system's c++filt is too old for the newer gcc's, 13.1 or 13.2 is acceptable

Reply:
Is it safe to rely on `/opt/compiler-explorer` though? For local instances Discovery would pull the system gcc.

Reply:
Actually it worked that way before, I should just revert it.

Reply:
this is not a file that people generally use locally